internal/operator-controller/applier/provider.go

```diff
@@ -70,18 +70,13 @@ func (r *RegistryV1ManifestProvider) Get(bundleFS fs.FS, ext *ocv1.ClusterExtens
 
 	if r.IsSingleOwnNamespaceEnabled {
 		bundleConfigBytes := extensionConfigBytes(ext)
-		// treat no config as empty to properly validate the configuration
-		// e.g. ensure that validation catches missing required fields
-		if bundleConfigBytes == nil {
-			bundleConfigBytes = []byte(`{}`)
-		}
 		bundleConfig, err := bundle.UnmarshalConfig(bundleConfigBytes, rv1, ext.Spec.Namespace)
 		if err != nil {
-			return nil, fmt.Errorf("invalid bundle configuration: %w", err)
+			return nil, fmt.Errorf("invalid ClusterExtension configuration: %w", err)
```

Comment:
at line 73 we unmarshall  bundle config, but then in the case of error we report now that ClusterExtension config is invalid? Should it stay `invalid bundle configuration`?